testing/trino-tests/src/test/java/io/trino/tests/AbstractTestEngineOnlyQueries.java

```diff
@@ -6026,34 +6028,18 @@ public void testShowFunctions()
     @Test
     public void testLargePivot()
     {
-        int columns = 254;
-
-        MaterializedResult result = computeActual(pivotQuery(columns));
+        MaterializedResult result = computeActual(pivotQuery(ARRAY_CONSTRUCTION_LIMIT));
         assertThat(result.getRowCount())
                 .as("row count")
-                .isEqualTo(columns);
+                .isEqualTo(ARRAY_CONSTRUCTION_LIMIT);
 
         MaterializedRow row = result.getMaterializedRows().get(0);
         assertThat(row.getFieldCount())
                 .as("field count")
-                .isEqualTo(columns + 2);
-    }
+                .isEqualTo(ARRAY_CONSTRUCTION_LIMIT + 2);
 
-    @Test
-    public void testPivotExceedingMaximumArraySize()
-    {
-        assertQueryFails(pivotQuery(255), "Too many arguments for array constructor");
```

Comment:
If you split the commit to two. First moving `pivotQuery` declaration here. The diff would be clener.

(please do not address that)